DeusEx/Classes/DXRKeys.uc

```diff
@@ -88,6 +207,7 @@ function MoveNanoKeys4()
         if ( SkipActorBase(k) ) continue;
 
         SetActorScale(k, 1.3);
+        k.ItemName = k.ItemName $ " (" $ k.Description $ ")";
```

Comment:
this is SO NICE, cause before you could only see the name of a key after picking it up, but if there was a datalink playing then it would cover that